<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotostationPlacerParamsBaseControlWidget</class>
 <widget class="QWidget" name="PhotostationPlacerParamsBaseControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhotostationPlacerParamsBase</string>
  </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>forceGps</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="forceGpsCheckBox">
       <property name="toolTip">
         <string>forceGps</string>
       </property>
       <property name="checked">
           <bool>true</bool>
       </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>speculativity</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="speculativitySpinBox">
       <property name="toolTip">
         <string>Enforce gps locations</string>
       </property>
       <property name="minimum">
        <number>-50000</number>
       </property>
       <property name="maximum">
        <number>50000</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>minimalInlierCount</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="minimalInlierCountSpinBox">
       <property name="toolTip">
         <string>minimalInlierCount</string>
       </property>
       <property name="minimum">
        <number>-50000</number>
       </property>
       <property name="maximum">
        <number>50000</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>32</number>
       </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>maximalFailureProbability</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="maximalFailureProbabilitySpinBox">
       <property name="toolTip">
         <string>maximalFailureProbability</string>
       </property>
       <property name="minimum">
        <double>-50000</double>
       </property>
       <property name="maximum">
        <double>50000</double>
       </property>
       <property name="singleStep">
        <double>1</double>
       </property>
       <property name="value">
        <double>0.15</double>
       </property>
     </widget>
    </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>forceGpsCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PhotostationPlacerParamsBaseControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>speculativitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PhotostationPlacerParamsBaseControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>minimalInlierCountSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PhotostationPlacerParamsBaseControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>maximalFailureProbabilitySpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PhotostationPlacerParamsBaseControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
 </customwidgets>
</ui>
